src/scf/mixing.jl

```diff
@@ -136,15 +140,15 @@ end
         δρ = devec(x)
         Jδρ = copy(δρ)
 
-        # Apply Kernel (just vc for RPA and (vc + fxc) if not RPA)
+        # Apply Kernel (just vc for RPA and (vc + K_{xc}) if not RPA)
```

Comment:
refer to the paper somewhere?

Reply:
Right at the top?

Reply:
Yeah for instance, or in the doc of hybrid (or both)